src/installer/pkg/sfx/Microsoft.NETCore.App/monocrossaot.sfxproj

```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.Build.NoTargets" DefaultTargets="BuildMonoCrossAllTargets">
```

Comment:
You don't need a default target here and just hook that target as `AfterTargets="Build"` to run after the build which is the default target.

Reply:
@jkoritzinsky just to make sure, the sfxproj SDK doesn't change the default target, right?

Reply:
You are correct. The SharedFX sdk still uses the Build target as the entry point.